itou/templates/employee_record/add.html

```diff
@@ -0,0 +1,67 @@
+{% extends "layout/base.html" %}
+{% load django_bootstrap5 %}
+{% load format_filters %}
+{% load str_filters %}
+{% load theme_inclusion %}
+
+{% block title %}Créer une fiche salarié {{ block.super }}{% endblock %}
+{% block content_title %}
+    <h1>Créer une fiche salarié - {{ request.current_organization.display_name }}</h1>
+{% endblock %}
+{% block content %}
+    <section class="s-section">
+        <div class="s-section__container container">
+            <div class="row">
+                <div class="col-12 col-md-9">
+                    <div class="c-stepper mb-3 mb-lg-5">
+                        <div class="progress progress--emploi">
+                            <div class="progress-bar progress-bar-{{ wizard|stepper_progress }}" role="progressbar" aria-valuenow="{{ wizard|stepper_progress }}" aria-valuemin="0" aria-valuemax="100">
+                            </div>
+                        </div>
+                        <p>
+                            <strong>Étape {{ wizard.steps.step1 }}</strong>/{{ wizard.steps.count }}
```

Comment:
```suggestion
                            <strong>Étape {{ wizard.steps.step }}</strong>/{{ wizard.steps.count }}
```
?

Reply:
Nope :).
> `step0` – The current step (zero-based).
> `step1` – The current step (one-based).

https://django-formtools.readthedocs.io/en/latest/wizard.html#creating-templates-for-the-forms